edx-platform/pearson-certiport-theme/lms/templates/header/header.html

```diff
@@ -22,9 +22,19 @@
 ## Provide a hook for themes to inject branding on top.
 <%block name="navigation_top" />
 
+## Add UI Toolkit components if using the Pattern Library
```

Comment:
This is not part of the Olive template. @sergivalero20

Reply:
@Squirrel18 Ok, I'll remove it.